lib/Interpreter/Interpreter.cpp

```diff
@@ -163,8 +171,15 @@ namespace cling {
     m_RawInputEnabled(false) {
 
     m_LLVMContext.reset(new llvm::LLVMContext);
+    if (!m_LLVMContext)
+      return;
     m_DyLibManager.reset(new DynamicLibraryManager(getOptions()));
+    if (!m_DyLibManager)
+      return;
+
     m_IncrParser.reset(new IncrementalParser(this, llvmdir));
+    if (!m_IncrParser || !m_IncrParser->isValid(false))
+      return;
```

Comment:
These commits add features where cling now has a mode to output files, to accomplish that without a lot of interface changes there needs to be a way to deal with the constructors understanding initialization of a full interpreter may not have occurred...more on that below.